x/crisis/module.go

```diff
@@ -164,3 +173,52 @@ func (am AppModule) EndBlock(ctx sdk.Context, _ abci.RequestEndBlock) []abci.Val
 	EndBlocker(ctx, *am.keeper)
 	return []abci.ValidatorUpdate{}
 }
+
+func init() {
+	appmodule.Register(
+		&modulev1.Module{},
+		appmodule.Provide(
+			provideModuleBasic,
+			provideModule,
+		),
+	)
+}
+
+type crisisInputs struct {
+	depinject.In
+
+	AppOpts    servertypes.AppOptions
+	Cdc        codec.Codec
+	Config     *modulev1.Module
+	Subspace   paramstypes.Subspace
+	BankKeeper types.SupplyKeeper `key:"cosmos.bank.v1.Keeper"`
```

Comment:
The `key:` annotation feature is no longer supported so these can be removed.

Reply:
thanks

Reply:
will do